internal/services/springcloud/client/client.go

```diff
@@ -14,6 +14,7 @@ type Client struct {
 	ApplicationAcceleratorClient *appplatform.ApplicationAcceleratorsClient
 	ApplicationLiveViewsClient   *appplatform.ApplicationLiveViewsClient
 	AppsClient                   *appplatform.AppsClient
+	ApmClient                    *appplatform.ApmsClient
```

Comment:
The SDK being used here (`tombuildsstuff/kermit`) is tech debt and should only be used where *absolutely* necessary (since we'll need to migrate off it later, as it's only a transitionary SDK) - it appears that this functionality is available in `hashicorp/go-azure-sdk`, as such there's no need to use `tombuildsstuff/kermit` here and we can use `hashicorp/go-azure-sdk` instead.

As such I've opened https://github.com/hashicorp/pandora/pull/3029 to add this API version to `hashicorp/go-azure-sdk` - once that's available can we update this to use `hashicorp/go-azure-sdk` once that's available?

Thanks!

Reply:
Got it!